examples/gnrc_minimal_xbee/Makefile

```diff
@@ -0,0 +1,44 @@
+# name of your application
+APPLICATION = gnrc_minimal_xbee
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= arduino-due
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD_INSUFFICIENT_MEMORY := chronos msb-430 msb-430h
+
+# Add the XBee module and nomac
+USEMODULE += xbee
+USEMODULE += gnrc_nomac
+
```

Comment:
@miri64 if this two modules are compiled the memory footprint will grow isn't it? If we merge it with the `gnrc_minimal` example and it's compiled for a board having it's own radio it won't conflict?